release-team/role-handbooks/docs/README.md

```diff
@@ -33,8 +33,8 @@ Responsibilities include:
 * Creating a dev branch used by contributors to target documentation updates for the upcoming release
 * Offering guidance to contributors about how to contribute new feature and enhancements documentation and working with contributors to modify existing docs to accurately represent any upcoming changes 
 * Providing weekly updates to the Release Team about the current state of release-bound docs
-* Mentoring Docs Lead Shadows throughout this process and empowering them with the knowledge needed to be future Docs Leads
-* Working with SIG-Docs to review documentation PRs according to the website [Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/) to ensure quality 
+* Mentoring Docs Shadows throughout this process and empowering them with the knowledge needed to be future Docs Leads
+* Working with Shadows to review documentation related PRs according to the [style guide] of (https://kubernetes.io/docs/contribute/style/style-guide/) website to ensure quality 
```

Comment:
Hey, the link is visually broken now:

![screenshot](https://github.com/kubernetes/sig-release/assets/695473/c4b75ab2-0961-46e0-8042-9f52a874d946)

Reply:
Uh-oh, my bad